packages/components/psammead-caption/src/index.jsx

```diff
@@ -14,19 +14,39 @@ import {
 import { getLongPrimer } from '@bbc/gel-foundations/typography';
 import { C_METAL } from '@bbc/psammead-styles/colours';
 import { scriptPropType } from '@bbc/gel-foundations/prop-types';
-import { getSansRegularItalic } from '@bbc/psammead-styles/font-styles';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
 
 const Caption = styled.figcaption`
-  ${({ script }) => script && getLongPrimer(script)};
-  ${({ service }) => getSansRegularItalic(service)}
+  ${({ script }) => script && getLongPrimer(script)}
+  ${({ service }) => getSansRegular(service)}
   color: ${C_METAL};
-  padding: ${GEL_SPACING} ${GEL_MARGIN_BELOW_400PX} 0;
+  padding: 0 ${GEL_MARGIN_BELOW_400PX} 0;
   width: 100%;
+  width: -webkit-calc(100% - ${GEL_SPACING});
+  width: -moz-calc(100% - ${GEL_SPACING});
+  width: -o-calc(100% - ${GEL_SPACING});
+  width: calc(100% - ${GEL_SPACING});
```

Comment:
- Thankfully we don't need to include browser prefixes for `calc` any more as it's widely supported amongst our browser base.
- I can see you have marked this PR for manual testing. 👍🏽 It's probably worth mentioning that IE11 has issues with certain configurations of `calc` (particularly rounding), so it's important these changes are tested before they're merged.

Reply:
> Super work, @HarryVerhoef. I've left a couple of comments below.

Thanks! I'll remove the prefixes 👍

Yeah I kept L24 as a fallback style, but if as you say calc is well supported among the browser base, I guess I could remove it?

Reply:
Awesome.
Yes, although Opera Mini doesn't support `calc`, `figcaption` is a block element and removing `width: 100%;` should not cause a regression.